docs/reference/elasticsearch/mapping-reference/search-as-you-type.md

```diff
@@ -143,33 +153,33 @@ More subfields enables more specific queries but increases index size.
 
 ## Parameters of the field type as a text field [general-params]
 
-The following parameters are accepted in a mapping for the `search_as_you_type` field due to its nature as a text-like field, and behave similarly to their behavior when configuring a field of the [`text`](/reference/elasticsearch/mapping-reference/text.md) data type. Unless otherwise noted, these options configure the root fields subfields in the same way.
+The following parameters are accepted in a mapping for the `search_as_you_type` field due to its nature as a text-like field, and behave similarly to their behavior when configuring a field of the [`text`](text.md) data type. Unless otherwise noted, these options configure the root fields subfields in the same way.
```

Comment:
```suggestion
The following parameters are accepted in a mapping for the `search_as_you_type` field due to its nature as a text-like field, and behave similarly to their behavior when configuring a field of the [`text`](/reference/elasticsearch/mapping-reference/text.md) data type. Unless otherwise noted, these options configure the root fields subfields in the same way.
```